packages/mux-player-react/src/index.tsx

```diff
@@ -70,6 +70,7 @@ export type MuxPlayerProps = {
   hotkeys?: string;
   nohotkeys?: boolean;
   defaultHiddenCaptions?: boolean;
+  disablePictureInPicture?: boolean;
```

Comment:
nitpick: Should be added to `MuxMediaPropTypes`, just above (which is part of the `MuxPlayerProps` type def, found just below)

Reply:
Fixed